test/govspeak_test.rb

```diff
@@ -827,4 +827,39 @@ class GovspeakTest < Minitest::Test
       |
     end
   end
+
+  test "inline attachment" do
+    attachment = OpenStruct.new(
+      content_id: "2b4d92f3-f8cd-4284-aaaa-25b3a640d26c",
+      id: 456,
+    )
+    govspeak = "[embed:attachments:inline:2b4d92f3-f8cd-4284-aaaa-25b3a640d26c]"
+    rendered = Govspeak::Document.new(govspeak, {attachments: attachment}).to_html
+    assert_match /<span id=\"attachment_456\" class=\"attachment-inline\">/, rendered
+  end
+
+  test "attachment" do
+    attachment = OpenStruct.new(
+      url: "www.test.com",
+      content_id: "2b4d92f3-f8cd-4284-aaaa-25b3a640d26c",
+      id: 123,
+      opendocument?: true,
+      order_url: "www.test.com",
+      price: 12.3,
+      csv?: true,
+      unnumbered_command_paper?: true,
+      isbn: 'isbn-123',
+    )
+    govspeak = "[embed:attachments:2b4d92f3-f8cd-4284-aaaa-25b3a640d26c]"
+    rendered = Govspeak::Document.new(govspeak, {attachments: attachment}).to_html
+    assert_match /<section class=\"attachment embedded\">/, rendered
```

Comment:
This assert and the ones below all trigger a ruby warning when you run the tests: http://stackoverflow.com/questions/5239805/what-triggers-the-ruby-warning-about-ambiguous-first-argument

can silence it by either using parenthesis on that method call or `%r{}` syntax